README.md

```diff
@@ -35,19 +35,19 @@ Hence why nobody is allowed to push commits directly to the core. All changes ar
 
 #### Authenticity
 
-Fixing and implementing missing blizzlike content is one of our priorities, and we can boast to offer the most content-complete open source emulator.
+Fixing and implementing missing blizzlike content is one of our priorities, and we can boast to offer the most content-complete open-source emulator.
 <!-- Not sure for this below -->
-<!-- We were the first open source emulator to have almost every dungeon and raid working. -->
+<!-- We were the first open-source emulator to have almost every dungeon and raid working. -->
 
 #### Modules
 
-Modules are essential to AzerothCore's success. Modules allows users to plug them in and out easily, and do not require to modify the core files. It also means users can keep pulling the git changes from the main repository and only develop their own modules.
+Modules are essential to AzerothCore's success. Modules allow users to plug them in and out easily, and do not require to modify the core files. It also means users can keep pulling the git changes from the main repository and only develop their modules.
 
 We have a lot of modules already made, some of them are very important and will ease your work:
 
 * [Transmogrification](https://github.com/azerothcore/mod-transmog)
 * [Eluna (lua engine) support](https://github.com/azerothcore/mod-eluna-lua-engine/ "Creator of Eluna is part of our core team")
-* Full list available in the AzerothCore catalogue (link at the end)
+* Full list available in the AzerothCore catalog (link at the end)
```

Comment:
what's wrong with `catalogue` ?

Reply:
I guess it is just me being too used with American English so I just auto changed it lol. can revert if British English is preferred

Reply:
yes, please :)